cord19-explorer/src/main/java/edu/asu/diging/cord19/explorer/web/ShowPublicationController.java

```diff
@@ -0,0 +1,29 @@
+package edu.asu.diging.cord19.explorer.web;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import edu.asu.diging.cord19.explorer.core.mongo.PublicationDao;
+
+@Controller
+public class ShowPublicationController {
+
+    @Autowired
+    private PublicationDao pubDao;
+
+    @RequestMapping("/publications")
+    public String get(Model model, Pageable pageable) {
+        model.addAttribute("pubs", pubDao.getPublications(pageable));
+        long pubCount = pubDao.getPublicationCount();
```

Comment:
doesn't the page that the getPublication method returns have the total number of elements?